interface-definitions/interfaces-tinc.xml.in

```diff
@@ -0,0 +1,451 @@
+            <properties>
+              <help>Ping Timeout(seconds,default:5)</help>
+            </properties>
+            <defaultValue>5</defaultValue>
+          </leafNode>
+          <leafNode name="priority-inheritance">
+            <properties>
+              <help>Enable Priority Inheritance</help>
+              <valueless/>
+            </properties>
+          </leafNode>
+          <leafNode name="private-keyfile">
+            <properties>
+              <help>Private Key File,This is the full path name of the RSA private key file that was generated by ‘tincd --generate-keys’. It must be a full path, not a relative directory(require)</help>
+            </properties>
+          </leafNode>
+          <leafNode name="hosts-dir">
+            <properties>
+              <help>The directory where the network interface stores host certificates(require)</help>
+            </properties>
+          </leafNode>
+          <leafNode name="priority">
+            <properties>
+              <help>Priority Option</help>
+              <completionHelp>
+                <list>low normal high</list>
+              </completionHelp>
+              <valueHelp>
+                <format>low</format>
+                <description>Low Priority</description>
+              </valueHelp>
+              <valueHelp>
+                <format>normal</format>
+                <description>normal Priority(default)</description>
+              </valueHelp>
+              <valueHelp>
+                <format>high</format>
+                <description>high Priority</description>
+              </valueHelp>
+              <constraint>
+                <regex>(low|normal|high)</regex>
+              </constraint>
+            </properties>
+            <defaultValue>normal</defaultValue>
+          </leafNode>
+          <node name="proxy">
+            <properties>
+              <help>Proxy settings</help>
+            </properties>
+            <children>
+              <leafNode name="type">
+                <properties>
+                  <help>Proxy Type</help>
+                  <completionHelp>
+                    <list>socks4 socks5 http exec</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>socks4</format>
+                    <description>socks4 Proxy</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>socks5</format>
+                    <description>socks4 Proxy</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>http</format>
+                    <description>http Proxy</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>exec</format>
+                    <description>Executes the given command which should set up the outgoing connection. The environment variables NAME, NODE, REMOTEADDRES and REMOTEPORT are available</description>
```

Comment:
"Command to execute on connection setup".
Environment variables and the like should be in the online docs, not in help strings.

Reply:
done